src/main/js/components/dropdowns/jumplists.js

```diff
@@ -17,7 +17,7 @@ function generateJumplistAccessors() {
     const dropdownChevron = document.createElement(
       isFirefox ? "span" : "button",
     );
-    dropdownChevron.className = "jenkins-menu-dropdown-chevron";
+    dropdownChevron.setAttribute("aria-label", "Open dropdown menu");
```

Comment:
I think it is wrong to replace the className and instead set the aria-label.
The className must stay and the aria-label must be added.

Reply:
this isn't ideal as its not localizeable, ideally the text would be taken from a jelly tag via a data attribute, that can be customizeable but has a default (that is localizeable)